MerlinAU.sh

```diff
@@ -233,6 +233,67 @@ _DoExit_()
    _ReleaseLock_ ; exit "$exitCode"
 }
 
+##------------------------------------------##
+## Modified by ExtremeFiretop [2024-May-21] ##
+##------------------------------------------##
+logo() {
+  echo -e "${YLWct}"
+  echo -e "      __  __           _ _               _    _ "
+  echo -e "     |  \/  |         | (_)         /\  | |  | |"
+  echo -e "     | \  / | ___ _ __| |_ _ __    /  \ | |  | |"
+  echo -e "     | |\/| |/ _ | '__| | | '_ \  / /\ \| |  | |"
+  echo -e "     | |  | |  __| |  | | | | | |/ ____ | |__| |"
+  echo -e "     |_|  |_|\___|_|  |_|_|_| |_/_/    \_\____/ ${GRNct}v${SCRIPT_VERSION}"
+  echo -e "${NOct}"
+}
+
+##---------------------------------------##
+## Added by ExtremeFiretop [2024-Jul-03] ##
+##---------------------------------------##
+_ShowAbout_(){
+    logo
+	cat <<EOF
+About
+  $SCRIPT_NAME is a tool for automating firmware updates on AsusWRT Merlin,
+  ensuring your router stays up-to-date with the latest features and security
+  patches. It simplifies the update process by automatically checking for,
+  downloading, and applying new firmware versions.
+  Developed by ExtremeFiretop and Martinski W.
+License
+  $SCRIPT_NAME is free to use under the GNU General Public License
+  version 3 (GPL-3.0) https://opensource.org/licenses/GPL-3.0
+Help & Support
+  https://www.snbforums.com/threads/merlinau-the-ultimate-firmware-auto-updater-addon.88577/
+Source code
+  https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router
+EOF
+	printf "\\n"
+    _DoExit_ 0
+}
+
+##---------------------------------------##
+## Added by ExtremeFiretop [2024-Jul-03] ##
+##---------------------------------------##
+_ShowHelp_(){
+    logo
+	cat <<EOF
+Available commands:
+  $SCRIPT_NAME about              explains functionality
+  $SCRIPT_NAME help               display available commands
+  $SCRIPT_NAME forceupdate        updates to latest version (force update)
+  $SCRIPT_NAME run_now            run update process on router
+  $SCRIPT_NAME processNodes       run update check on nodes
+  $SCRIPT_NAME develop            switch to development branch
+  $SCRIPT_NAME stable             switch to stable branch
+  $SCRIPT_NAME uninstall          uninstalls script
+  $SCRIPT_NAME addCronJob         add the MerlinAU cron job
+  $SCRIPT_NAME postRebootRun      add the post-update reboot cron job
+  $SCRIPT_NAME postUpdateEmail    Send post-update email
```

Comment:
The last 3 call parameters (**addCronJob**, **postRebootRun**, **postUpdateEmail**) are intended **only** for internal use to handle specific events and are not meant to be exposed to users for CLI calls. Those lines should be removed from the "`_ShowHelp_()`"  function.

Reply:
No worries! I can remove them from the help function :)

I was just including them all while I was at it, I wasn't sure if we should or shouldn't include the original commands so good to know we can keep those hidden.

Reply:
Done here: https://github.com/ExtremeFiretop/MerlinAutoUpdate-Router/pull/260/commits/65eb51d4f4e633c47ba101ee2292ebf1fbb27f4f